<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SalesListForm</class>
 <widget class="QWidget" name="SalesListForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#040609;</string>
  </property>
  <widget class="QTableView" name="tableView_sales">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>1101</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_deleteSale">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>800</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Dana&quot;;
color: rgb(0, 0, 0);
background-color: rgb(132, 0, 0);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>حذف فاکتور</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_code">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>800</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>800</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_quantity">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>800</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_price">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>800</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_total">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>800</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>800</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
border-radius: 10px;
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_menu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>780</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Dana&quot;;
background-color: rgb(255, 38, 49);
color: rgb(0, 0, 0);
border-radius: 10px;
</string>
   </property>
   <property name="text">
    <string>بازگشت به منو</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_widget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>30</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 20pt &quot;Dana&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>فاکتورهای فروش</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>781</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;
background-color: rgb(85, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>زمان ثبت</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>781</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;
background-color: rgb(85, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>کد کالا</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>781</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;
background-color: rgb(85, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>نام کالا</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>781</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;
background-color: rgb(85, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>تعداد</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>781</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;
background-color: rgb(85, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>قیمت </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>781</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 700 12pt &quot;Dana&quot;;
background-color: rgb(85, 255, 127);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>مجموع</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>tableView_sales</zorder>
  <zorder>pushButton_deleteSale</zorder>
  <zorder>pushButton_menu</zorder>
  <zorder>label_widget</zorder>
  <zorder>label</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>label_10</zorder>
  <zorder>lineEdit_date</zorder>
  <zorder>lineEdit_total</zorder>
  <zorder>lineEdit_price</zorder>
  <zorder>lineEdit_quantity</zorder>
  <zorder>lineEdit_name</zorder>
  <zorder>lineEdit_code</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
